docs/terminology.rst

```diff
@@ -0,0 +1,28 @@
+.. _terminology:
+
+Terminology
+===========
+
+
+.. csv-table:: Core Concepts
+    :header: "Name", "Description"
+    :widths: 30, 60
+
+    **SpecFile**,"A ``SpecFile`` is a YAML file that buildtest interprets when generating the test. The SpecFile may
```

Comment:
Hmm, this is confusing for several reasons. First, calling it a "SpecFile" is just really general, how is anyone supposed to know it's relevant to build test? To follow convention of Dockerfile, Makefile, Singularity recipe, I think the file should be called something specific to buildtest? A Buildspec? And then "Spec" is confusing, because it's really referring to a block (a test) in the file. Why not just refer to this as a Build Test? Basiclly:

 - a Buildspec is a file that buildtest interprets
 - A build test defined as a block in the file that is validated by a Buildspec Schema
 - a Buildspec Schema is a JSON file defining valid / invalid key value pairs and formatting for the file
 - And then you can still use Schema Library and the rest those make sense.

But overall I think the first three are a bit confusing and we need to rethink!

Reply:
I like ``BuildSpec`` that is a unique name so if we adopt this name then ``--config`` could be ``--buildspec``. I think its important the names be meaningful and unambiguous for end-users and developers.

Reply:
**A build test defined as a block in the file that is validated by a Buildspec Schema** is this a definition part of ``BuildSpec``. Should i remove the keyword ``Spec``.

Reply:
You shouldn't use my words verbatim, i was just giving an example.

Reply:
I updated can you check 2nd revision. I removed ``Spec`` that was addressing one of more test block. That is now just part of ``BuildSpec`` to avoid confusion.